Linux/buildenv.mk

```diff
@@ -28,7 +28,7 @@ ifeq ($(MITIGATION_INDIRECT), 1)
     MITIGATION_CFLAGS += -mindirect-branch-register
 endif
 ifeq ($(MITIGATION_RET), 1)
-    CC_VERSION := $(shell $(CC) -dumpversion)
+    CC_VERSION := $(shell $(CC) -dumpversion | cut -d"." -f1)
```

Comment:
It needs to align to the SGX SDK code:
https://github.com/intel/linux-sgx/blob/2ee53db4e8fd25437a817612d3bcb94b66a28373/buildenv.mk#L227

Reply:
I've reverted this change as the following comparison uses `expr` instead of the `-ge` operator.